x-pack/plugins/ml/public/application/routing/breadcrumbs.ts

```diff
@@ -106,7 +106,7 @@ export const getBreadcrumbWithUrlForApp = (
 ): EuiBreadcrumb => {
   return {
     text: breadcrumbs[breadcrumbName].text,
-    ...(navigateToPath && basePath
+    ...(navigateToPath
```

Comment:
For the pages under Model Management, we need the extra level for the Nodes Overview / Trained Models breadcrumbs:

![image](https://user-images.githubusercontent.com/7405507/151552703-283feb45-1646-4ecc-8fd9-2b4518c123c6.png)

Reply:
Confirmed fixed by https://github.com/elastic/kibana/pull/123805/commits/a5717c005c1857502ec3a80259246f09de668852